build_tools/cmake/iree_llvm.cmake

```diff
@@ -93,6 +94,11 @@ macro(iree_llvm_configure_installed)
   if(NOT LLVM_EXTERNAL_LIT)
     set(LLVM_EXTERNAL_LIT "${IREE_SOURCE_DIR}/third_party/llvm-project/llvm/utils/lit/lit.py")
   endif()
+
+  set(IREE_LLVM_LINK_BINARY "${LLVM_INSTALL_DIR}/llvm/bin/llvm-link")
+  set(IREE_CLANG_BINARY "${LLVM_INSTALL_DIR}/llvm/bin/clang")
```

Comment:
These might want to append `CMAKE_EXECUTABLE_SUFFIX` for Windows compatibility... I'm not sure.

Reply:
I think you're right!